Adds New Alert Levels, Updates SOLFED Marshals, Updates Alert Screens (#4992)
NovaSector/NovaSector: config/nova/config_nova.txt

ALERT_VIOLET_DOWNTO  A major medical emergency has developed. Non-medical personnel are required to obey all relevant instructions from medical staff.
ALERT_DELTA_UPTO Destruction of the station is imminent. All crew are instructed to obey all instructions given by heads of staff. Any violations of these orders can be punished by death. This is not a drill.
ALERT_DELTA_DOWNTO Destruction of the station is still imminent. All crew are instructed to obey all instructions given by heads of staff. Any violations of these orders can be punished by death. This is not a drill.
ALERT_GAMMA_UPTO The Solar Federation has placed this system under Gamma Alert status. This galactic system is facing a ZK-Class Reality Failure Scenario. Security Personnel is authorized full access to lethal equipment to enforce Martial Law. Failure to follow emergency procedures is punishable by death. This is not a drill.

> ALERT_GAMMA_DOWNTO The Solar Federation has released Military Command of the station, the Solar Federation has detected and declared a ZK-Class Reality Failure Scenario. Security Personnel is authorized full access to lethal equipment to enforce Martial Law. Failure to follow emergency procedures is punishable by death. This is not a drill.

Reviewer: ngl it feels a bit odd that the 'reality failiure' is less severe than generic military control. Not important in this PR tho
~~honestly this whole alert feels kind of hamfisted. "ZK-Class Reality Failiure"? Its just gibberish... it should be a more generic "System-Wide threat". So the scale goes from Red (crew), Delta (station), Gamma (starsystem), SolFed (galactic sector). Then of course Epsilon is just 'control of all company assets'~~

Author: I was thinking yeah we could do that where solfed is declaring a massive threat of some kind is present and localized martial law is in effect

Reviewer: (ignore the deleted comment im dumb and misread what i was commenting on)

Reviewer: Actually reading it I'm realizing that this should probably be made more generic, since it's super unlikely that NT control would go to solfed, just for solfed to release it. Odds are if we DO switch to this alert, NT's going to go straight to it.

Also no clue how best to word that, but just something like "Control has been relinquished to Station Command" in place of specifically solfed releasing control. Type of sentance that covers both NT and Sol releasing it, if that makes sense

Author: For right now it works, but I can do more with it tomorrow

> ALERT_EPSILON_UPTO Nanotrasen Central Command has declared the station under full CentCom authority. All CentCom personnel and representatives now hold absolute control of the station and its assets. Crew cooperation is compulsory. Have a Secure Day!

Reviewer: With this being an up-to, potentially from the SolFed alert, might be good to quickly specify that it's above SolFed control

No clue how to best put that though. If I knew more legal-ese I'd say maybe something like "Per [SolFed-NT agreement of some sort], all CentCom Personnel and representatives blah blah blah". Just something to put the idea of 'sol, maybe inadvertently through legal stuff, relinquished control to NT'

ALERT_DELTA_UPTO Destruction of the station is imminent. All crew are instructed to obey all instructions given by heads of staff. Any violations of these orders can be punished by death. This is not a drill.
ALERT_DELTA_DOWNTO Destruction of the station is still imminent. All crew are instructed to obey all instructions given by heads of staff. Any violations of these orders can be punished by death. This is not a drill.
ALERT_GAMMA The Solar Federation has placed this system under Gamma Alert status. This galactic system is facing a ZK-Class Reality Failure Scenario. Security Personnel is authorized full access to lethal equipment to enforce Martial Law. Failure to follow emergency procedures is punishable by death. This is not a drill.

> ALERT_EPSILON Central Command has deemed the station under centcomm authority, centcomm personnel are now in full authority of the station. your cooperation is compulsory.

Reviewer: TG actually has a tool in place (`tools\ci\check_grep.sh`) that's meant to catch this, but it might only hit maps - CC's abbreviation has 1 M and both C's capitalized, `CentCom`

Also the alert wording should probably just be a) more corporate, and b) more assertive.

_(Also IDK if calling it "Nanotrasen Central Command" would be better here, that part's just a spitball)_
_(Also also I think adding the NT classic Secure Day catchphrase, or their official motto (Nanotrasen: A Better Tomorrow, Today.) or the like at the bottom would be fun too but again just a spitball)_
```suggestion
ALERT_EPSILON Nanotrasen Central Command has declared the station under full CentCom authority. All CentCom personnel and representatives now hold absolute control of the station and its assets. Crew cooperation is compulsory. Have a Secure Day!
```

Author: I think it would go hard to be honest!

> ALERT_FEDERAL The Solar Federation has placed this system under its full command, This galactic system is now under Military Law, comply and await for official instructions.

Reviewer: This one is more formatting and phrasing; SolFed would 100% get straight to the point as opposed to CentCom corpo slang.
However, Galactic **systems** are too wide a scope, sticking to the Sector helps narrow down how close this threat really is.

_(The ending still feels off to me, doesn't flow right, like it's not addressing anyone specifically? idk how to improve that without it getting wordy and rambling tho and its 430am LOL)_

```suggestion
ALERT_FEDERAL The Solar Federation has placed this galactic sector under its full command. All star systems within this sector are now under Military Law. Comply with all official SolFed instruction.
```

Author: you forgot an s

Reviewer: proof that it I am not lying about the time being 430am LOL
